xclim/indices/_agro.py

```diff
@@ -1094,16 +1090,15 @@ def _get_rain_season(pram):
 
 @declare_units(
     pr="[precipitation]",
-    pr_cal="[precipitation]",
     params="[]",
 )
 def standardized_precipitation_index(
     pr: xarray.DataArray,
-    pr_cal: Quantified | None = None,
     freq: str | None = "MS",
     window: int = 1,
     dist: str = "gamma",
     method: str = "APP",
+    fitkwargs: dict = {},
```

Comment:
using an empty dictionary as a default is dangerous

https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Reply:
Good call! I tend to use a pattern that resembles the following:
```
my_func(arg: str, kwarg: dict | None = None):
    if kwarg3 is None:
        kwarg3 = dict{}
```